### PR TITLE
Use Redisson Session Manager for Docker Redis

### PR DESCRIPTION
**Pull request by sjuarezgx**
Issue:107025

### CODE REVIEW

**Review by anaiberta — CHANGES_REQUESTED** [review]
Why do we have jars as part of the service? Can we define them as Maven Dependencies?

File: src/Docker/createpackage.msbuild [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="CreatePackage"  ToolsVersion="4.0">
 	<PropertyGroup>
 		<GXDeployFileProject>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\..\..\web\$(ProjectName).gxdproj'))</GXDeployFileProject>
+		<RedissonLibs>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\context\tomcat\lib'))</RedissonLibs>
```
Comment (anaiberta) [inline src/Docker/createpackage.msbuild:4]: Please, think of use / as path separator. \ is not cross-platform (we will have to change them to be used at GeneXus Next Edition.

File: src/Docker/redis/Java/build.gradle [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+configurations {
+    redissonConfiguration 
+}
+
+ext {
+    defaultRedissonLibsDirectory = "tomcat/lib"
+    tomcatVersion = project.hasProperty('TomcatVersion') ? project.getProperty('TomcatVersion') : 'TOMCAT_10_1'
+    redissonLibsDirectory = project.hasProperty('RedissonLibsDirectory') ? project.getProperty('RedissonLibsDirectory') : 'defaultRedissonLibsDirectory'
```
Comment (anaiberta) [inline src/Docker/redis/Java/build.gradle:14]: Look out with the 'defaultRedissonLibsDirectory', is a string, not the value of defaultRedissonLibsDirectory.

Comment (sjuarezgx) [reply src/Docker/redis/Java/build.gradle]: Fixed.

**Review by anaiberta — APPROVED** [review]
There are a lot of \ that need to be changed for / to be cross-platform. But now you know that we have to change all them, we can do it in a specific issue.

**Pull request merged**